__tests__/app.test.js

```diff
@@ -149,9 +146,18 @@ describe('/api/articles/:article_id', () => {
     });
 });
 
+describe('GET /api/articles/:article_id/?comments=true' , () => {
+    test('status:200 return the comment count of article' , () => {
+        return request(app)
+        .get('/api/articles/1/?comments=true')
+        .expect(200)
```

Comment:
This task should work without a query added, so you're sort of changing what you had previously. You can either change the original test to now include a check for comment_count or write a separate one. It should just go to /api/articles/1 though